code/modules/jobs/job_types/janitor.dm

```diff
@@ -4,7 +4,7 @@
 	department_head = list("Head of Personnel")
 	department_flag = CIVILIAN
 	faction = "Station"
-	total_positions = 2
+	total_positions = 6
```

Comment:
this may be a overkill, why not try 4 for now?

Reply:
even on low pop the halls are a mess